src/main/haskell/language-kore/src/Data/Kore/ASTPrettyPrint.hs

```diff
@@ -505,7 +497,23 @@ instance
     prettyPrint flags (VariablePattern p)      =
         writeOneFieldStruct flags "VariablePattern" p
 
-instance PrettyPrint (pat variable)
+instance (PrettyPrint (t c a b)) => PrettyPrint (Rotate31 t a b c) where
+    prettyPrint flags (Rotate31 x) =
+        writeOneFieldStruct flags "Rotate31" x
+
+instance PrettyPrint CommonKorePattern where
+    prettyPrint flags korePattern =
+        writeOneFieldStructK flags "Fix"
+        $ writeOneFieldStructK NeedsParentheses "UnifiedPattern"
+        $ case getUnifiedPattern (unFix korePattern) of
+            UnifiedMeta p ->
+                writeOneFieldStructK NeedsParentheses "UnifiedMeta"
+                $ writeOneFieldStruct NeedsParentheses "Rotate31" (unRotate31 p)
```

Comment:
why `writeOneFieldStruct` and not `prettyPrint`?

Reply:
Because, for some reason, the instance for Rotate31 was not working, and I forgot to remove it;  did it now.